db/schema.rb

```diff
@@ -415,21 +415,22 @@
     t.integer "ecstat4"
     t.integer "ecstat5"
     t.integer "ecstat6"
+    t.integer "disabled"
+    t.integer "wheel_value_check"
     t.integer "pregyrha"
     t.integer "pregla"
     t.integer "pregghb"
     t.integer "pregother"
-    t.integer "disabled"
-    t.integer "wheel_value_check"
-    t.integer "hhregres"
-    t.integer "hhregresstill"
     t.string "ppostcode_full"
     t.boolean "is_previous_la_inferred"
     t.integer "ppcodenk"
     t.string "ppostc1"
     t.string "ppostc2"
     t.string "prevloc"
     t.boolean "previous_la_known"
+    t.integer "hhregres"
+    t.integer "hhregresstill"
+    t.integer "prevten"
```

Comment:
Idk if it would work for you, but often if I also run the migrations in test env, that cleans up the schema file for me (rake db:migrate RAILS_ENV=test)